tpot/gp_deap.py

```diff
@@ -103,7 +103,8 @@ def varOr(population, toolbox, lambda_, cxpb, mutpb):
 
 
 def eaMuPlusLambda(population, toolbox, mu, lambda_, cxpb, mutpb, ngen, pbar,
-                   stats=None, halloffame=None, verbose=0, max_time_mins=None):
+                   stats=None, halloffame=None, verbose=0, max_time_mins=None,
+                   periodic_pipeline_saver=None):
```

Comment:
Please refactor `periodic_pipeline_saver` to `periodic_pipeline_saving_fn` or something more descriptive. Please also document it in the docstrings.